packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationPage.tsx

```diff
@@ -18,7 +18,7 @@ import { OrganizationProfileAvatarUploader } from '../OrganizationProfile/Organi
 
 export const CreateOrganizationPage = withCardStateProvider(() => {
   const title = localizationKeys('createOrganization.title');
-  const subtitle = localizationKeys('createOrganization.subtitle');
+  const inviteTitle = localizationKeys('organizationProfile.invitePage.title');
```

Comment:
Can you elaborate on this change?

Reply:
We need to have this as the title in the second step of the create-organization page.

Reply:
Should we keep this as a localization key to avoid a breaking change(and not use it like in the example)?

Reply:
Deprecated the old key